viam-orb-slam3_test.go

```diff
@@ -481,15 +413,14 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 func createSLAMService(
 	t *testing.T,
 	attrCfg *slamConfig.AttrConfig,
-	model string,
```

Comment:
We no longer need to specify the model type (it used to be "orbslamv3" or "fake_orbslamv3") because everything is orbslam